_inc/client/apps/style.scss

```diff
@@ -160,50 +160,55 @@
 	}
 }
 
-.jp-landing-apps__feature-col {
+.jp-landing-apps__feature {
 
 	@include breakpoint( ">660px" ) {
-		flex-basis: 50%;
 
-		@include breakpoint( ">660px" ) {
```

Comment:
There totally was in master:
 
![image](https://cloud.githubusercontent.com/assets/1123119/22032366/6ad44448-dca1-11e6-8932-bb874b1a2b7a.png)

Nice fix!